src/structures/textChannel.ts

```diff
@@ -270,17 +280,21 @@ export class GuildTextChannel extends TextChannel {
     this.nsfw = data.nsfw ?? this.nsfw
     this.parentID = data.parent_id ?? this.parentID
     this.topic = data.topic ?? this.topic
-    this.rateLimit = data.rate_limit_per_user ?? this.rateLimit
+    this.slowmode = data.rate_limit_per_user ?? this.slowmode
   }
 
+  /** Edit the Guild Text Channel */
   async edit(
     options?: ModifyGuildTextChannelOption
   ): Promise<GuildTextChannel> {
     const body: ModifyGuildTextChannelPayload = {
       name: options?.name,
       position: options?.position,
       permission_overwrites: options?.permissionOverwrites,
-      parent_id: options?.parentID
+      parent_id: options?.parentID,
+      nsfw: options?.nsfw,
+      topic: options?.topic,
+      rate_limit_per_user: options?.rateLimitPerUser
```

Comment:
I think we should change this `rate_limit_per_user` to `slowmode` too

Reply:
You mean options.rateLimitPerUser? rate_limit_per_user is a name in API payload can't change lol

Reply:
oh sorry but yeah the option one

Reply:
Done.